pytorch_lightning/trainer/evaluation_loop.py

```diff
@@ -90,14 +90,14 @@ def on_evaluation_start(self, *args, **kwargs):
         else:
             self.trainer.call_hook('on_validation_start', *args, **kwargs)
 
-    def on_evaluation_model_eval(self, *args, **kwargs):
+    def on_evaluation_model_eval(self, *_, **__):
```

Comment:
I've never seen this. Is this a new convention? :)

Reply:
I saw it already in our codebase https://github.com/PyTorchLightning/pytorch-lightning/blob/ce9179591dc3414aa0ff59007ea46f30fa9ddbee/pytorch_lightning/trainer/connectors/logger_connector/epoch_result_store.py#L140

Reply:
Definitely not new, here is a topic about it from 8 years ago :laughing: https://stackoverflow.com/a/11486174